### PR TITLE
Added Symfony User Group Munich talk 2015-09-29

### PR DESCRIPTION
**Pull request by PhilippHToner**

### CODE REVIEW

File: src/AppBundle/Resources/data/block.yml [hunk 1]
```diff
@@ -71,3 +71,10 @@ static:
             publishEndDate: "2015-10-16 12:15"
             link: http://berlin2015.live.symfony.com/programm/#session-014
             body: Sulu and the other guys - Wie Sulu mit anderen Applikationen zusammen spielt.
+    symfony-usergroup-meetup-munich-2015-09-29:
+        class: AppBundle\Document\TalkBlock
+        properties:
+            title: Symfony User Group Munich
+            publishEndDate: "2015-09-29 18:30"
+            link: http://www.meetup.com/de/sfugmunich/events/224029031/
+            body: Symfony CMF and Sulu talk at the PHP user group Munich
```
Comment (dbu) [inline src/AppBundle/Resources/data/block.yml]: should this not say "Symfony user group" instead of "PHP user group"?

Comment (PhilippHToner) [inline src/AppBundle/Resources/data/block.yml]: that's correct sorry. I force pushed the correct one.

**Comment by dbu** [comment]
:+1: and good luck with the talk!

**Comment by PhilippHToner** [comment]
thank you :)

**Pull request merged**